test/lib/server/services/log/LogService.test.js

```diff
@@ -66,4 +66,32 @@ module.exports = () => {
         expect(rootLogsFirstPage.logs[0].id).to.equal(1);
         expect(rootLogsSecondPage.logs[0].id).to.equal(5);
     });
+
+    it('Should successfully return child logs of their parent log', async () => {
+        const logWithChildrenId = 117;
+        const { childLogs } = await logService.getChildLogsByParentId(logWithChildrenId);
+
+        childLogs.forEach((log) => expect(log.parentLogId === logWithChildrenId));
+    });
+
+    it('Should successfully return an empty array if parent has no children', async () => {
+        const logWithoutChildrenId = 122;
+        const { childLogs } = await logService.getChildLogsByParentId(logWithoutChildrenId);
+
+        expect(childLogs).to.be.empty;
+    });
+
+    it('Should successfully return correct count when children are found', async () => {
+        const logWithChildrenId = 117;
+        const { count, childLogs } = await logService.getChildLogsByParentId(logWithChildrenId);
+
+        expect(childLogs.length).to.equal(count);
+    });
```

Comment:
I get count from findAndCountAll, so I thought this was simply the amount of logs since no limit or offset are given for these child logs. Am I mistaken?

Reply:
Ah ok I did not notice. Then I think you should not return the count and simply use `findAll` in here, and you should put a hard-coded length in the test.